packages/clay-alert/docs/index.js

```diff
@@ -27,11 +27,25 @@ render(<Component />);`;
 const alertImportsCode = `import React from 'react';
 import ClayAlert from '@clayui/alert';`;
 
+const AlertJSPCode = `<clay:alert
```

Comment:
Do we need to also provide a snippet of how to import the clay taglibs? Or would that be too redundant? We could possibly put that into on a different JSP specific page on how to get started with the taglibs.

Reply:
I think that we should have a JSP specific page, a central area instead of polluting the examples

Reply:
Should go in the snippets, basically:

```
<%@ taglib uri="http://liferay.com/tld/clay" prefix="clay" %>
```

Reply:
@jbalsas  Is our pattern in DXP is to include the taglib inside `init.jsp` once and then use `clay:*` in whichever JSP imports `init.jsp`?

If so, maybe we don't want to leave the impression upon the user they need to import the taglib whenever they use a new `clay:*` tag. Probably even, the module they're working on already has this import.

Reply:
I don't think this is an issue. DPX features some heavy SF rules and such trivial issue will be caught instantly and never forgotten. 

Let's not mud things here based on DXP conventions. I'm sure engineers will know how to adapt an import line in one jsp.

Reply:
Alright, sure! It is one line of code after all so I'm sure it will be fine 👍